faker_edu/__init__.py

```diff
@@ -136,6 +136,35 @@
     'Teaching Assistant',
 ]
 
+SPORTS = [
+    "Baseball",
+    "Basketball",
+    "Bowling",
+    "Cricket",
+    "Cross country",
+    "Curling",
+    "Fencing",
+    "Field hockey",
+    "Football",
+    "Golf",
+    "Gymnastics",
+    "Ice hockey",
+    "Lacrosse",
+    "Rifle",
+    "Rugby",
+    "Rowing",
```

Comment:
To keep lists easy to search we have a test that ensure alphabetical order.  Rowing should come before Rubgy.